[Tiny PR] Add default schedule to scheduling guide (#2831)
astronomer/docs: learn/cross-dag-dependencies.md

A common use case for this implementation is when an upstream DAG fetches new testing data for a machine learning pipeline, runs and tests a model, and publishes the model's prediction. In case of the model underperforming, the TriggerDagRunOperator is used to start a separate DAG that retrains the model while the upstream DAG waits. Once the model is retrained and tested by the downstream DAG, the upstream DAG resumes and publishes the new model's results.

> The [schedule](scheduling-in-airflow.md) of the downstream DAG is independent of the runs triggered by the TriggerDagRunOperator. For a DAG to only run based on the TriggerDagRunOperator, set the `schedule` parameter to `None`. Note that the dependent DAG must be unpaused for it to be triggered by the TriggerDagRunOperator.

Reviewer: ```suggestion
The [schedule](scheduling-in-airflow.md) of the downstream DAG is independent of the runs triggered by the TriggerDagRunOperator. To run a DAG solely with the TriggerDagRunOperator, set the DAG's `schedule` parameter to `None`. Note that the dependent DAG must be unpaused to get triggered.
```
